libs/authorizenet/src/drivers/authorize-net.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DaffPaymentDriver, DaffPaymentTransformer } from '@daffodil/checkout';
+
+import { DaffAuthorizeNetPaymentService } from './authorize-net.service';
+import { DaffAuthorizeNetTransformerService } from './transformers/authorize-net-transformer.service';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ]
+})
+export class DaffAuthorizeNetDriverModule {
+  static forRoot(): ModuleWithProviders {
```

Comment:
What happens when there are two providers for the `DaffPaymentDriver` should this be a `multi` provider?

Reply:
yea, I changed it